### DISCUSSION TITLE
Hurl: open all web protocols

### DISCUSSION THREAD

**Discussion by Clemens-Ratte-Polle** [U-C-S/Hurl#72] [Ideas]
idea:
Hurl could open all web protocols like mailto: etc. with Outlook, Thunderbird.

**Comment by U-C-S** [comment]
should be possible, but I must test each of them and edge cases slowly, so one at a time. 

I shall look into `mailto` some time

**Comment by eidylon** [comment]
I would say this should be an option in the settings though. Most people only use one email client, so there's not AS much need for switching email clients I don't think, from my experience working in IT support and dealing with clients/users. 

Not to say there may not be some people who could use this, but it would definitely be a much smaller user segment. 
So I'd say either have `Additional Protocols` be an option in the settings, or at worst wait to implement that until we have the option to save default handlers, so that we'd be able to set up the default handler for other protocols in the settings, and not have to be prompted every time. .... or both; an additional setting, AND default handlers.

**Reply by Jay-o-Way** [reply]
> Most people only use one email client

Possibly, but not always. Scenario: I use my Outlook account for (what's the word?) "business" things, like things I usually have to pay for, haha. Google is for my "personal" stuff. My Outlook account is connected to my computer and my Google account is connected to my phone. Therefore a `mailto` link may open the Windows Outlook app, or Gmail (which is a PWA). So yeah, same idea applies to e-mails.

**Comment by sjain882** [comment]
To expand on the single email client argument, I think it could be useful for more apps, such as `steam://` etc, and another major use case for this is here: https://github.com/U-C-S/Hurl/issues/111
